internal/builders/generic/README.md

```diff
@@ -41,29 +51,55 @@ output:
 $ sha256sum artifact1 artifact2 ... | base64 -w0
 ```
 
-After you have encoded your digest, add a new job to call the
-`slsa-github-generator` reusable workflow. Here's an example of what it might
-look like all together.
+After you have encoded your digest, add a new job to call the reusable workflow.
+
+```yaml
+provenance:
+  permissions:
+    actions: read # Needed for detection of GitHub Actions environment.
+    id-token: write # Needed for provenance signing and ID
+    contents: read # Needed for API access
+  uses: slsa-framework/slsa-github-generator/.github/workflows/generator_generic_slsa3.yml@main
```

Comment:
let's not do @main?

Reply:
Yep. That's a holdover from the previous docs. We actually have tags now!